spec/ohloh_scm/parser/cvs_parser_spec.rb

```diff
@@ -0,0 +1,77 @@
+      assert_equal 'pizzandre', revisions[1].committer_name
+      assert_equal Time.utc(2005, 0o7, 25, 17, 11, 6), revisions[1].committer_date
+      assert_equal 'Addin UNL file with using example-', revisions[1].message
+    end
+
+    # One file with several revisions
+    it 'must test multiple revisions' do
+      revisions = OhlohScm::CvsParser.parse File.read(FIXTURES_DIR + '/multiple_revisions.rlog')
+
+      # There are 9 revisions in the rlog, but some of them are close together with the same message.
+      # Therefore we bin them together into only 7 revisions.
+      assert_equal 7, revisions.size
+
+      assert_equal '2005/07/15 11:53:30', revisions[0].token
+      assert_equal 'httpd', revisions[0].committer_name
+      assert_equal 'Initial data for the intelliglue project', revisions[0].message
+
+      assert_equal '2005/07/15 16:40:17', revisions[1].token
+      assert_equal 'pizzandre', revisions[1].committer_name
+      assert_equal '*** empty log message ***', revisions[1].message
+
+      assert_equal '2005/07/26 20:35:13', revisions[5].token
+      assert_equal 'pizzandre', revisions[5].committer_name
+      assert_equal "Issue number:\nObtained from:\nSubmitted by:\nReviewed by:\nAdding current milestones-",
+                   revisions[5].message
+
+      assert_equal '2005/07/26 20:39:16', revisions[6].token
+      assert_equal 'pizzandre', revisions[6].committer_name
+      assert_equal "Issue number:\nObtained from:\nSubmitted by:\nReviewed by:\nCompleting and fixing milestones texts",
+                   revisions[6].message
+    end
+
+    # A file is created and modified on the branch, then merged to the trunk, then deleted from the branch.
+    # From the trunk's point of view, we should see only the merge event.
+    it 'must test file created on branch as seen from trunk' do
+      revisions = OhlohScm::CvsParser.parse File.read(FIXTURES_DIR + '/file_created_on_branch.rlog')
+      assert_equal 1, revisions.size
+      assert_equal 'merged new_file.rb from branch onto the HEAD', revisions[0].message
+    end
+
+    # A file is created on the vender branch. This causes a simultaneous checkin on HEAD
+    # with a different message ('Initial revision') but same committer_name name and timestamp.
+    # We should only pick up one of these checkins.
+    it 'must test simultaneous checkins' do
+      revisions = OhlohScm::CvsParser.parse File.read(FIXTURES_DIR + '/simultaneous_checkins.rlog')
+      assert_equal 1, revisions.size
+      assert_equal 'Initial revision', revisions[0].message
+    end
+
+    # Two different authors check in with two different messages at the exact same moment.
+    # How this happens is a mystery, but I have seen it in rlogs.
+    # We arbitrarily choose the first one if so.
+    it 'must test simultaneous checkins_2' do
+      revisions = OhlohScm::CvsParser.parse File.read(FIXTURES_DIR + '/simultaneous_checkins_2.rlog')
+      assert_equal 1, revisions.size
```

Comment:
Run `sed -Ei 's/^(\s*)assert_equal ([^,]+),\s([^,#]+)$/\1\3.must_equal \2/' spec/...` on spec files to convert assert_equal to must_equal.